frontend/app/handlers.ts

```diff
@@ -1,19 +1,47 @@
+import {
+    crypto_pwhash_str,
+    crypto_pwhash_str_verify,
+    crypto_pwhash_OPSLIMIT_INTERACTIVE,
+    crypto_pwhash_MEMLIMIT_INTERACTIVE,
+} from "libsodium-wrappers-sumo";
+
 // Make API call to server to check if password was found in breached dataset
 export const checkSecurity = async (password: string) => {
     try {
-        const response = await fetch("http://localhost:18080/intersection", {
-            method: "POST",
-            mode: "cors",
-            headers: {
-                "Access-Control-Allow-Headers": "*", // cors setting
-                "Content-Type": "application/json"
-            },
-            body: password
-        })
+        // hash the password
+        const passwordHash = crypto_pwhash_str(
+            password,
+            crypto_pwhash_OPSLIMIT_INTERACTIVE,
```

Comment:
**question:** is this because we are using "interactive, online operations"? So we are using the INTERACTIVE because it somehow dynamically checks the size of the password?